src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -230,6 +245,56 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
             await RespondAsync(request, ct, input);
         }
 
+        public Task<SendBlobResult> SendBlobAsync(byte[] data, CancellationToken ct) {
```

Comment:
Semantically - yes, but under the hood, do we use different sockets or message formats for those messages?